compat/mingw.c

```diff
@@ -3587,8 +3587,29 @@ static void setup_windows_environment(void)
 			struct strbuf buf = STRBUF_INIT;
 			strbuf_addstr(&buf, tmp);
 			if ((tmp = getenv("HOMEPATH"))) {
+				/*
+				 * Since Vista, HOMEDRIVE/HOMEPATH are set to Windows\System32
+				 * for secondary logons/services instead of being unset.
+				 * We don't want to (and likely can't) save stuff there
+				 * so ignore them in such a case.
+				 */
+				wchar_t w_system_dir[MAX_PATH];
```

Comment:
Nowhere else in the code base do we use that `w_` prefix. Please match the existing coding style, for consistency.

Reply:
Sorry, I was wrong. It's not nowhere that we use it, it is highly unconventional in our source code.

I'd much rather see you using a name like `system32_path`.